cairo/tests/ethereum/cancun/test_fork.py

```diff
@@ -821,8 +787,10 @@ def test_apply_body(self, cairo_run, data, withdrawals: Tuple[Withdrawal, ...]):
 
     @pytest.mark.parametrize(
         "zkpi_path",
-        list(Path("data/1/eels").glob("*.json")),
-        ids=[x.stem for x in Path("data/1/eels").glob("*.json")],
+        [
+            Path("test_data/21688509.json"),
+            *[Path(p) for p in glob("data/inputs/1/*.json")],
```

Comment:
this makes the test very long, maybe not such a good idea

Reply:
for sure, we should select blocks in test_data only imo. data/** should not be commited